src/utils/mapsheet.ts

```diff
@@ -117,11 +121,11 @@ export const MapSheet = {
     };
     // 1:500 has X/Y is 3 digits not 2
     if (out.gridSize === 500) {
-      out.y = Number(match[3]?.slice(0, 3));
-      out.x = Number(match[3]?.slice(3));
+      out.y = Number(match?.groups?.['tileId']?.slice(0, 3));
+      out.x = Number(match?.groups?.['tileId']?.slice(3));
     } else {
-      out.y = Number(match[3]?.slice(0, 2));
-      out.x = Number(match[3]?.slice(2));
+      out.y = Number(match?.groups?.['tileId']?.slice(0, 2));
+      out.x = Number(match?.groups?.['tileId']?.slice(2));
```

Comment:
I just realised we need a case for the 50k imagery as this returns, as `out.x` and `out.y` will be NaN for the 50k imagery.